src/RuleSet/Sets/PERSet.php

```diff
@@ -19,14 +19,23 @@
 /**
  * @internal
  *
- * Last updated to PER Coding Style v1.0.0.
+ * Last updated to PER Coding Style v1.1.0.
```

Comment:
I believe that even if we not fully implement v1.1, we may already take changes from those PR and make them life. And then, over time, we can fulfil more rules of 1.1 to the PER set.
What do you think, @jrmajor ?

Reply:
I agree, v1.1 hasn't been released yet though :)

Reply:
We can't, as no version of PER includes these rules, they're only on `dev-master`. Unless they tag v1.1, they can revert or change newly updated sections. As soon as they tag it, I'm planning to do exactly what you propose.